README.md

```diff
@@ -7,61 +7,168 @@
 
 * Explain that by making parts of their code *conditional*, they can execute different pieces of code based on certain conditions.
 * Explain that an *if* statement will execute a set of statements **only** if that condition is *true*. 
+* Explain that an *if* statement can provide an alternative set of statements, known as an *else clause*, for situations when the *if* condition is *false*.
+* Create/chain multiple *if-else* statements together to consider additional clauses:
+
+
+
+## Outline / Notes
+
+In the previous lesson we covered Boolean values (namely ````true```` and ````false````) along with the various operations we can perform that result in a Boolean value (namely ````<, ==, <=, >, >=, !=````).
+For example:
+
+````Swift
+3 < 5    // false
+12 >= 12  // true
+let greedoFiredFirst  // true
+! greedoFiredFirst  // false
+````
```

Comment:
Perfect!! This is great, touching back on what they know and reinforcing that is a great approach.

The only change here I would ass is having the ! operator hugging the variable and declaring the `greedoFiredFirst` constant to have a value of `true`. Or maybe instead of declaring the constant with `let`, we just write something like this:

```swift
greedoFiredFirst  // true
!greedoFiredFirst  // false
```

or this.. (I leave that decision up to you).

```swift
let greedoFiredFirst = true  // true
!greedoFiredFirst  // false
```